ftests/nuxeo-server-gatling-tests/pom.xml

```diff
@@ -0,0 +1,268 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo.ftests</groupId>
+    <artifactId>nuxeo-ftests</artifactId>
+    <version>11.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>nuxeo-server-gatling-tests</artifactId>
+  <name>Nuxeo Server Gatling Bench</name>
+  <description>Gatling bench on Nuxeo Server</description>
+  <packaging>jar</packaging>
```

Comment:
we could remove it since it's the default value

Reply:
I think @troger tried to switch to pom during the `ftests` creation without success.

I'm removing the gatling packaging for now and let this discussion open.